src/ui/OmniboxResultList/OmniboxResultList.ts

```diff
@@ -276,6 +283,9 @@ export class OmniboxResultList extends ResultList implements IComponentBindings
   }
 
   private handlePopulateOmnibox(args: IPopulateOmniboxEventArgs) {
+    if (this.options.numberOfPreviewResults > 0) {
```

Comment:
I would store this condition in a getter, `shouldShowPreviewResults` and then use it here and on line 320.